d2/src/main/java/com/linkedin/d2/balancer/dualread/DualReadLoadBalancerMonitor.java

```diff
@@ -97,14 +97,7 @@ public void reportData(String propertyName, T property, String propertyVersion,
                   fromNewLb ? "New" : "Old", propertyClassName, propertyName, existingEntry._version,
                   propertyVersion, property);
 
-          if (isUriProp)
-          {
-            LOG.debug(msg);
-          }
```

Comment:
This already logging to debug level. No need to further change it. And it's critical to warn Service and Cluster properties mismatch, so we only log to debug level for Uri properties.